test/functional/browser.study.api.js

```diff
@@ -1206,4 +1209,12 @@ describe("PUBLIC API `browser.study` (not specific to any add-on background logi
       it("log level works?");
     });
   });
+};
+
+describe("PUBLIC API `browser.study` (studyType: shield)", function() {
+  publicApiTests.bind(this)("shield");
+});
+
+describe("PUBLIC API `browser.study` (studyType: pioneer)", function() {
+  publicApiTests.bind(this)("pioneer");
```

Comment:
I usually see this written as `publicApiTests.call(this, "pioneer")`. I don't have any reason to prefer one or the other though.